plugins/rapid7_insightvm_cloud/bin/icon_rapid7_insightvm_cloud

```diff
@@ -6,7 +6,7 @@ from sys import argv
 
 Name = "Rapid7 InsightVM Cloud"
 Vendor = "rapid7"
-Version = "3.1.0"
+Version = "3.1.1"
```

Comment:
Should that be the PATCH version or MINOR version bump if there is a new action?